### DISCUSSION TITLE
自動投稿BOTについて

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#243] [要望・アイデア・不具合]
最近、同じタイトルで毎日自動投稿しているBOTがいるのですが、Qiitaの利用方法として問題ないでしょうか？
ランキング自動投稿BOTなどもありますがひとつの記事を自動更新していてそんなに気にならないのですが、情報変化量の少ない記事を毎日自動投稿するのはQiitaガイドラインの行動指針から外れている気がしてしまいます。
自動投稿に関するガイドラインを作るか、利用規約の禁止事項に追加してはいかがでしょうか？

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご意見ありがとうございます🙏
ガイドラインや利用規約のアップデートは検討中です💪
また、特定のアカウントをご指摘いただく際は、[サポートフォーム](https://support.qiita.com/hc/ja/requests/new) よりお問い合わせいただけると幸いです🙏🙇💦

**Reply by shiracamus** [reply]
該当リンクは削除しておきました。

ガイドラインのアップデートを検討しておられるとのことなので、ついでに、、、
外部URLをひとつ書いてあるだけの記事も散見されるので、「リンクだけを書くのではなく、自分が何を学んだのか、つまづいた点や克服方法など、自分の体験・知識を記録・共有し、参考情報としてリンクを記載しましょう」といったガイドライン記載も検討してはいかがでしょうか？
URLだけを書いて説明などを書かないのは禁止行為にしてもいいかもしれません。リンク集は有用なので線引きが難しいですけど。

**Reply by tanaka-asami415** [reply]
@shiracamus 
ご対応いただきありがとうございます🙇

はい！具体的な事例を明記するかは判断が難しいところですが、
今回いただいたご意見も、ガイドラインアップデートの際に反映できるよう検討させていただきます💪
